src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/specify-when-alerts-create-incidents.mdx

```diff
@@ -100,54 +96,62 @@ Each option for incident preference has its own advantages. You can select any o
 
 <figcaption>
-  When you select the default (**By policy**) as your incident preference, it will group all condition violations into the same incident.
+  **[one.newrelic.com](https://one.newrelic.com) > Alert & AI > Policies > (select a policy):** The selected policy page shows how alerts rolls up incidents for alert notifications and UI details. (Default is **By policy**). To choose a different option for this policy, click **Incident preference**.
 </figcaption>
 
-By default, a single incident record will be created for each alert policy. If additional conditions are violated by any target after the incident opens, the violations are added to the same incident. Only one incident appears on your **Incidents** indexes for the selected policy.
 
-Selecting the **By policy** preference is the most general option. All condition violations for every target will be accumulated into a single incident record.
 
-## Example: By condition [#preference-condition]
+## By policy (default) [#preference-policy]
+
+![A diagram showing how the By policy incident preference works](./images/by-policy.svg "A diagram showing how the By policy incident preference works")
 
-![Alert policy incident preferences](./images/example-condition-incident.png "Alert policy incident preferences")
+<figcaption>
+  When you select the default (**By policy**) as your incident preference, it will group all condition violations into the same incident.
+</figcaption>
+
+If additional conditions are violated by any target after the incident opens, the violations are added to the same incident. Only one incident appears on your Incidents indexes for the selected policy.
+ 
+Choose the By policy preference if your policy and its conditions have a focused scope (like monitoring one entity). Learn more about [incident preference best practices](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#incident-practices).
+
+## By condition [#preference-condition]
+
+![A diagram showing how the By condition incident preference works](./images/by-condition.svg "A diagram showing how the By condition incident preference works")
 
 <figcaption>
   When you select **By condition** as your incident preference, it will group all violations by any target for a specific condition into separate incidents.
 </figcaption>
 
-Selecting the **By condition** option is useful, for example, when you want an individual incident record to focus on a specific condition. If the policy has multiple conditions, separate incident records for each condition will appear on your **Incidents** indexes.
+Choose the By condition option when you want an individual incident record to focus on a specific condition. If the policy has multiple conditions, separate incident records for each condition will appear on your Incidents indexes.
```

Comment:
I'm not sure why you removed the bolding.... But you should put it back, I think.